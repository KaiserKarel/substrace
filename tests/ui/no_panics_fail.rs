#![allow(enable_singlepass_benchmarks)]

fn main() {}