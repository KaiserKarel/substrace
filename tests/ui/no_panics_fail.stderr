error: substrace: clippy must be configured to warn or deny about any panicking code
  --> $DIR/no_panics_fail.rs:1:1
   |
LL | #![allow(enable_singlepass_benchmarks)]
   | ^
   |
   = note: `-D panics` implied by `-D warnings`
help: insert attributes at the root of the crate
   |
LL + #![warn(
LL +     clippy::disallowed_methods,
LL +     clippy::indexing_slicing,
LL +     clippy::todo,
LL +     clippy::unwrap_used,
LL +     clippy::panic,
LL + )]
LL ~ #![allow(enable_singlepass_benchmarks)]
   |

error: aborting due to previous error

